src/UpdateHTML.ts

```diff
@@ -306,6 +306,10 @@ export const revealStuff = () => {
         document.getElementById("toggleautoresearch").style.display = "block" :
         document.getElementById("toggleautoresearch").style.display = "none";
 
+    player.shopUpgrades.obtainiumAuto > 0 && player.cubeUpgrades[9] > 0 ? //Auto Research Shop Purchase Mode
+        document.getElementById("toggleautoresearchmode").style.display = "block" :
+        document.getElementById("toggleautoresearchmode").style.display = "none";
```

Comment:
```suggestion
    document.getElementById("toggleautoresearchmode").style.display = player.shopUpgrades.obtainiumAuto > 0 && player.cubeUpgrades[9] > 0 ? //Auto Research Shop Purchase Mode
        ? 'block'
        : 'none';
```

Reply:
Again, consistency

Reply:
consistency isn't always the best (I changed a few of those blocks above, so there isn't really any consistency whatsoever anyways)